pyproject.toml

```diff
@@ -8,7 +8,7 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "scipy-openblas64"
-version = "0.3.26.0.4"
+version = "0.3.26.0.5"
```

Comment:
@mattip I don't know enough about the project to know why this is required, I just did what your PR template told me and bumped it. As this is simply changing a script that only runs in CI is this version bump actually needed? If not, please let me know and I'll remove this.

Reply:
That version bump is needed to distinguish the wheels uploaded to PyPI which does not allow overriding an already-uploaded file.